packages/zosjobs/src/doc/input/ICancelJobParms.ts

```diff
@@ -33,8 +33,8 @@ export interface ICancelJobParms {
 
     /**
      * version of the cancel request
-     * @type {string}
+     * @type {boolean}
      * @memberof ICancelJobParms
      */
-    version?: string;
+    version?: boolean;
```

Comment:
Apologies for not noticing sooner, but isn't this considered a breaking change?

Reply:
I agree this seems like a breaking change. At least in the LTS branch, we should leave the version property as type "string" same as it was before.